metricbeat/module/system/process/_meta/docs.asciidoc

```diff
@@ -11,6 +11,14 @@ This metricset is available on:
 - Windows
 
 [float]
+=== Cache CmdLine
+
+This metricset caches the command line args for a running process. This means if you alter
+the command line for a process while this metricset is running, these changes are not detected.
+This feature can disabled by adding
+`process.cmdline.cache: false` to the system module configuration.
```

Comment:
We should add this option also to the full config file: https://github.com/elastic/beats/blob/master/metricbeat/module/system/_meta/config.full.yml (don't forget make update ;-) )